app/services/finance/ecf/output_calculator.rb

```diff
@@ -3,249 +3,16 @@
 module Finance
   module ECF
     class OutputCalculator
```

Comment:
I think from the spike the output calculator isn't shared, maybe we can keep that same assumption as there is nothing to share between mentor and ECTs? And we have different ECF and mentor output calculators?